tests/nlu/test_evaluation.py

```diff
@@ -369,6 +370,7 @@ async def test_run_evaluation(unpacked_trained_moodbot_path: Text):
     assert result.get("intent_evaluation")
 
 
+@pytest.mark.trains_model
```

Comment:
`test_run_cv_evaluation` and `test_run_cv_evaluation_with_response_selector` also train (even multiple times)

Reply:
I guess not through `nlu.train`?

Reply:
No 😬  It trains through the `Trainer`.